### ISSUE TITLE
Phaseque: Rack 2 Compatibility

### ISSUE THREAD

**Issue by zezic** [zezic/ZZC#84]
Need to port Phaseque to Rack 2.

**Comment by nay-seven** [comment]
good news ! ;-)

**Comment by trevormeier** [comment]
Looking forward to these in Rack 2!

**Comment by nay-seven** [comment]
Hello, 3 months later, no news, 
are you ok ?

**Comment by zezic** [comment]
Hi, @nay-seven. I'm OK, however the situation around me (and in the world) is not quite stable at the moment. But anyways, to the good, a few days ago I fixed the undo/redo history handling for Phaseque and it seems that it's only left to repair the parameter values updating when patterns are switched.

Also I'm thinking about moving Phaseque to open source after slight code clean up.

**Comment by nay-seven** [comment]
hey, really happy all is ok for you !
And yes this world is clearly going more and more crazy ! ;-(
but happily, we have the music.
good news for the modules, and if you need beta tester ( Mac mojave & catalina here) count with me !

**Comment by Setkias** [comment]
Great to hear you're ok.
Hearing you're thinking about making Phaseque open source sounds very interesting.
Count me in as a beta tester (Win/Lin).

**Comment by pgatt** [comment]
Count me in as a beta tester too. On Mac here.

**Comment by pgatt** [comment]
@zezic Any updates?

**Comment by VegaDeftwing** [comment]
@zezic Any update on this?

**Comment by nay-seven** [comment]
yes, could be really cool to inform users who support you by buying your modules ?
i need to use vcv 1.1.6 to use Phaseque, but could be easier in rack 2

**Comment by trevormeier** [comment]
Any news on this? Phaseque looks rad! I’d love to buy it but I’m a VCV2 user

**Comment by zezic** [comment]
I posted with the update regarding the Phasueque @ Rack 2 port in #93.
TLDR;: I hope to complete the port on this weekend _...again._

**Comment by nay-seven** [comment]
Really happy for this news !!

**Comment by sounds99** [comment]
I know it’s been a year but so many are still excited for 2. Phaseque.
